node/resolvers/index.ts

```diff
@@ -259,6 +261,15 @@ export const resolvers = {
           if (getOrganizationById?.paymentTerms) {
             settings.paymentTerms = getOrganizationById.paymentTerms
           }
+
+          // if organization status is "on-hold" or "inactive", remove "can-checkout" permission
+          if (getOrganizationById?.status !== 'active') {
```

Comment:
Shouldn't it be (getOrganizationById?.status === 'active') ?

Reply:
If it's active, we use the normal logic. If it's not active, we filter out the "can checkout" permission.